pontis-ui/package-lock.json

```diff
@@ -1,9582 +0,0 @@
-{
```

Comment:
I think that it's not needed because I'm using `yarn` for everything now. 
There was a short period where I was using `npm` a while ago, so I think this file is left over from then.

Reply:
Ok. But looks like yarn.lock is also deleted?

Reply:
Seems to me like it's just changed?